### DISCUSSION TITLE
Problems while trying to boot through bootable USB

### DISCUSSION THREAD

**Discussion by p3nox** [mocaccinoOS/mocaccino#70] [Q&A] [answered]
Hi, I'm struggling to get the mocaccinoOS Gnome to boot.

I'm trying to make it to work for about 2 hour now.

the following erro is happening, right after the system tries to mount "/tmp/mnt/image"

` mount: /tmp/mnt/image:wrong fs type, bad option, bad superblock on /dev/loop0, missing codepage or helper program, or other error`

`mount failed (squashfs)`

`Overlay data from device /dev/sdc has been merged.`

`/sbin/init in rootfs not found, dropping to emergency shell `

After that, the system just open the "sh" for doing commands and stays on that.

I've already made a checksum, and the ISO is perfect.

I have a xeon x3440 as processor, a gtx 750ti as graphic card, and a p7h55d-m EVO as motherboard (It not supports UEFI). I'm using the latest available bios for the motherboard.

I've tried to boot using Ventoy, and after the same error, i tried to make a bootable usb with the BalenaEtcher. Same error after BalenaEtcher.

My USB drive is working 100%, and I've tried with 2 different ISOs, both of them, with the perfect checksum. None of them worked.

I'm trying to make it work, but I'm totally new to Gentoo based distros, so I'm trying to get any possible help to use the mocaccinoOS.

**Comment by joostruis** [comment]
Hi there.
Just to exclude a problem here. Is there another bootable Linux USB drive inserted in the system at the same time?

**Reply by p3nox** [reply]
there is none, aside the mocaccinoOS bootable drive, my keyboard and my mouse.

**Comment by joostruis** [comment]
Did you download the image here?
https://github.com/mocaccinoOS/mocaccino/releases/tag/v0.1.0-beta03

**Reply by p3nox** [reply]
Yes I've downloaded from there. And as said, i downloaded 2 times, each one with a checksum test. both tests having a positive result.

**Comment by joostruis** [comment, marked as answer]
Very odd issue then. I even tested creating my USB here with Etcher and that worked fine for me. 
It looks like there is an issue with the USB filesystem itself. Have you tried using gnome-disk-utility or even dd?

**Reply by p3nox** [reply]
Nope. I'm gonna try it and return with the results.

**Reply by p3nox** [reply]
now i'm writing from inside of MocaccinoOs.

After trying to make a bootable usb through GnomeDiskUtility and dd, both of them returned the same error, i tried to make a checksum again. Apparently after trying to make a bootable usb through etcher, it broke my Iso, considering that the checksum was wrong. i've just extracted again, and the checksum was good again (i've just used the same .tar.xz that i used before, so i guess the etcher just broke my Iso).

Then i made another bootable usb using the GnomeDiskUtility, and everything work perfectly(until now at least).

Thanks for the help :)
